types/react-rnd/react-rnd-tests.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import * as Rnd from 'react-rnd';
```

Comment:
We should be using an `import Rnd = require("react-rnd"` import statement here. Users who want can use `allowSyntheticDefaultImports` for this package (or potentially `esModuleInterop` if they're using something like Browserify).

Reply:
Ok, I will be fix it.